Dockerfile

```diff
@@ -1,21 +1,53 @@
 ############################################################
 ## Dockerfile to build the ga4gh server on mod_wsgi-express
 ## Configurable to use a local dataset
-## Based on mod_wsgi-docker
-## Results of this build are available from Dockerhub as afirth/ga4gh_server_apache:prod
 ############################################################
-FROM grahamdumpleton/mod-wsgi-docker:python-2.7-onbuild
+FROM ubuntu
 
-# File Author / Maintainer
-MAINTAINER Alastair Firth
+# Originally created by Steve Hershman GitHub @hershman
```

Comment:
I don't think it matters if we use the code or not, he still contributed to the development of it. You could argue that git tracks this perfectly well, so we don't need to ACK Steve either. I'm fine with either deleting the ACK entirely, or ACKing both Steve and Alastair.